src/sfdxFlags.ts

```diff
@@ -167,7 +161,7 @@ function buildEnum<T>(options: flags.Enum<T>): flags.Discriminated<flags.Enum<T>
     options: options.options,
     description: options.description,
     longDescription: options.longDescription,
-  };
+  } as flags.Discriminated<flags.Enum<T>>;
```

Comment:
I don't understand sfdxComand flags well enough to know why this is required.  Is the return type wrong?  Should flags.Enum not have a parse method, or it should return something else?

Having to make assertions that compiler doesn't believe usually means something's not quite right across all cases.

Reply:
it might even indicate that Enum from oclifFlags is wrong?